README.md

```diff
@@ -18,23 +17,29 @@ Kick-off your project with this default boilerplate. This starter ships with the
 
     ```shell
     # Clone this repo into your machine
-    npx degit vtex-sites/base.store my-awesome-store
+    npx degit vtex-sites/base.store awesome.store
     ```
 
 1.  **Install dependencies**
 
     Install dependencies with yarn
 
     ```shell
-    cd my-awesome-store/
+    cd awesome.store/
     yarn
     ```
 
-2.  **Setup env vars.**
+2.  **Setup env vars**
 
-    Chose a store by changing the `vtex.env` file and adding your own custom account name.
+    Choose the ecommerce platform provider of your choice in the `vtex.env` file and set the corresponding options. For instance, to connect to the VTEX platform on the store `fashioneurope`:
 
-3.  **Start developing.**
+    ```
+    GATSBY_STORE_ID=fashioneurope
+    GATSBY_VTEX_ENVIRONMENT=vtexcommercestable
+    GATSBY_VTEX_IO_WORKSPACE=master
```

Comment:
What do you think about explaining each env var? For example, GATSBY_STORE_ID is the store accounts name on VTEX.

Reply:
I believe it's not necessary because the env name is explicit. A good code is one that doesn't need comments to be understood, this is a case in point.

Reply:
It's written before the example that this works for  fahsioneurope at the stable environment. Is this enough?